client/state/selectors/get-active-discount.js

```diff
@@ -59,13 +42,5 @@ export default ( state ) => {
 		return null;
 	}
 
-	const activeVariation = discount.variations
-		? discount.variations[ abtest( discount.abTestName ) ]
-		: null;
-
-	if ( ! activeVariation ) {
-		return discount;
-	}
-
-	return composeActiveDiscount( discount, activeVariation );
+	return discount;
```

Comment:
In my PR I was shortening this to `return activeDiscounts.find( ... ) ?? null`.

Reply:
Nice, I've borrowed this in 45a754fdee6b5f935552459e1fd1e937587aead8